docs/Other/marketplace_webhooks.md

```diff
@@ -774,6 +774,14 @@ title: Customer Delete Event
 
 The spend change request webhook is called every time a spend change is requested for one of the vendor's products.
 
+Details:

```

Comment:
Could you add an explanation as to how the system picks what currency to do the conversion into, where it gets the rate, and why there is a conversion at all. I assume it has something to do with the SMB paying in a different currency than the vendor's system consumes variable spend in.

Reply:
Sure, I'll add more description to the docs. We must always convert non-USD values to USD because while partners can pay in USD, AUD, or CAD for digital ads and potentially other products, Vendasta only pays vendors in USD.